iTjuana/itj-library
# ✨ Feature book page

### Description of the Change


See more details of a book on its own page.

### Verification Process


1. Run the UI
2. Go to catalog
3. Click a book
4. See the data

### Applicable Issues


- Closes #38 

### Release Notes


Create Book page and some changes to API
